### ISSUE TITLE
Support the libaom library?

### ISSUE THREAD

**Issue by fangzb** [arthenica/ffmpeg-kit-test#63]
Does it support the libaom library? My requirement is to encode the video into av1 format

**Comment by tanersener** [comment]
If you build `ffmpeg-kit` with `libaom` support then you can use the test applications to test the build.

**Comment by fangzb** [comment]
> If you build `ffmpeg-kit` with `libaom` support then you can use the test applications to test the build.

Thanks for your reply. What should I do next?  just "enable --libaom" and build?

**Comment by tanersener** [comment]
Yes, build a new `ffmpeg-kit` binary, run one of the top level build scripts (e.g. android.sh, ios.sh, tvos.sh, macos.sh) and add the `--enable-libaom` flag.

**Comment by fangzb** [comment]
> Yes, build a new `ffmpeg-kit` binary, run one of the top level build scripts (e.g. android.sh, ios.sh, tvos.sh, macos.sh) and add the `--enable-libaom` flag.

Understood, I will give it a try and come to a conclusion later. Thank you

**Comment by fangzb** [comment]
> > Yes, build a new `ffmpeg-kit` binary, run one of the top level build scripts (e.g. android.sh, ios.sh, tvos.sh, macos.sh) and add the `--enable-libaom` flag.
> 
> Understood, I will give it a try and come to a conclusion later. Thank you

Following your instructions, after rebuilding the build, libaom is now supported, but it is very very slow ... It takes an hour to transcode a 20MB video。Have you tried libsvtav1 or other encode lib?

**Comment by tanersener** [comment]
Yes, `libaom` is slow, so we don't build it by default. We also tried other `AV1` encoders, but ran into problems with them."
